model/bootstrap/node_info.go

```diff
@@ -56,6 +63,31 @@ type NodeMachineAccountInfo struct {
 	HashAlgorithm sdkcrypto.HashAlgorithm
 }
 
+func (info NodeMachineAccountInfo) FlowAddress() flow.Address {
+	// trim 0x-prefix if present
+	addr := info.Address
+	if strings.ToLower(addr[:2]) == "0x" {
+		addr = addr[2:]
+	}
+	return flow.HexToAddress(addr)
+}
+
+func (info NodeMachineAccountInfo) PrivateKey() (crypto.PrivateKey, error) {
+	sk, err := crypto.DecodePrivateKey(info.SigningAlgorithm, info.EncodedPrivateKey)
+	if err != nil {
+		return nil, fmt.Errorf("could not decode machine account private key: %w", err)
+	}
+	return sk, nil
+}
+
+func (info NodeMachineAccountInfo) MustPrivateKey() crypto.PrivateKey {
+	sk, err := info.PrivateKey()
+	if err != nil {
+		panic(err)
```

Comment:
It will have `"could not decode machine account private key:` from the `PrivateKey` method, and panicking will print the stack trace, giving the line number.